tests/python/test_ast_refactor.py

```diff
@@ -794,6 +797,9 @@ def foo(x: tc.template()) -> tc.i32:
 
 @test_utils.test(require=ti.extension.assertion, debug=True, gdb_trigger=False)
 def test_assert_message():
+    if sys.platform == "linux" and os.uname()[4] in ["arm64", "aarch64"]:
+        pytest.skip("assert not currently supported on linux arm64 or aarch64")
```

Comment:
I would suggest adding a helper to detect whether the current arch is ARM on linux (local to unit test).

Moreover, you should prefer static `pytest.mark.skip` over runtime `pytest.skip` everytime the condition can be resolved statically, which is almost always the case, here included.